cobalt/updater/network_fetcher.h

```diff
@@ -41,6 +41,9 @@ namespace updater {
 typedef enum UrlFetcherType {
   kUrlFetcherTypePostRequest,
   kUrlFetcherTypeDownloadToFile,
+#if defined(IN_MEMORY_UPDATES)
```

Comment:
I highly recommend _not adding_ a compile-time definition for this. Is there a downside to include all this #ifdef code by default in everything ?

Reply:
> I kept it for now so we can easily verify that the customizations are made cleanly

Can we do this verification with a run-time, rather than compile-time switch ? A config value, command line flag or such ?

Reply:
I'd then go ahead and change those to `USE_COBALT_CUSTOMIZATIONS` right away at least here in net code, so that they are always on for all build configs. Gets better test coverage across the board.

Reply:
Done for `net/url_request` - sounds like you're ok with me replacing/removing the `IN_MEMORY_UPDATES` checks in other parts of the code in a quick follow-up PR, which is my preference.